packages/newman-reporter-allure/src/index.ts

```diff
@@ -333,8 +336,32 @@ class AllureReporter {
     const requestData = this.currentRunningItem?.pmItem.requestData;
     const requestDataURL = requestData && `${requestData.method} - ${requestData.url}`;
 
+    if (requestData?.header && requestData?.header?.count() > 0) {
+      const headers = requestData.header
```

Comment:
I think, we can write headers as `JSON` file, and it will be much more convenient to read and use them

Reply:
Ok, changed to JSON.